pom.xml

```diff
@@ -39,6 +39,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>1.8</java.version>
 		<lombok.version>1.18.4</lombok.version>
+		<spring.boot.version>1.5.6.RELEASE</spring.boot.version>
```

Comment:
Any reason to have this as variable? It is used only once right

Reply:
Right now the same version is used across the parent pom, http-service pom and cache adapter pom, maybe it should just be set once here in the parent and removed in the other two poms?

Reply:
yeah

Reply:
After rechecking my pull request, that is actually what I had already done, the variable is just defined once in the parent and used throughout the project 
<img width="661" alt="Screen Shot 2019-05-12 at 11 44 58 AM" src="https://user-images.githubusercontent.com/16943514/57585924-aea83680-74ab-11e9-9a5c-416a8f1a8f94.png">